src/renderers/dom/shared/__tests__/ReactDOMComponent-test.js

```diff
@@ -1611,10 +1615,10 @@ describe('ReactDOMComponent', () => {
       ReactTestUtils.renderIntoDocument(<input type="text" onclick="1" />);
       expectDev(console.error.calls.count()).toBe(2);
       expectDev(normalizeCodeLocInfo(console.error.calls.argsFor(0)[0])).toBe(
-        'Warning: Unknown DOM property class. Did you mean className?\n    in div (at **)',
+        'Warning: Invalid DOM property `class`. Did you mean `className`?\n    in div (at **)',
```

Comment:
I reworded Unknown to invalid since we will technically allow any attribute, and `class` is the invalid form of `className`.